### ISSUE TITLE
Incomplete voiceover integration 

### ISSUE THREAD

**Issue by bernhard-eiling** [veeso/ios-media-player#1]
The controls of the player are insufficiently covered by voiceover:

Labels displaying the current time and the total time of a video are spelled out only by the numbers shown. e.g. "three, forty" and not "Three minutes and forty seconds".

The tracker bar only provides numerical values with no context e.g. "nine percent". We think "video progress: nine percent" or a similar message would be more appropriate.

There seems to be two accessibilty elements covering the whole video frame.

**Comment by ov1d1u** [comment]
Hi,

Thanks for your suggestions. Veeso 2.0.28 comes with a better Voiceover integration which also includes your suggestions.

Regards,
Ovidiu
